DDDEastAnglia.Tests/Controllers/SessionControllerTests.cs

```diff
@@ -73,29 +69,63 @@ public void CannotBeginToDeleteASession_WhenTheSessionBelongsToAnotherUser()
         public void CannotConfirmToDeleteASession_WhenTheSessionBelongsToAnotherUser()
         {
             var session = new Session { SessionId = 1, SpeakerUserName = "bob" };
-            var controller = SessionControllerFactory.Create(session);
+            var controller = new SessionControllerBuilder().Updating(session).Build();
 
             var result = controller.DeleteConfirmed("fred", 1);
 
             Assert.That(result, Is.InstanceOf<HttpUnauthorizedResult>());
         }
 
-        private static class SessionControllerFactory
+        [TestFixture]
+        public class EmailTests
```

Comment:
I find this being a nested class a bit clumsy, especially as we don't get the benefit of the fluent naming.